From: Tomas Henzl <thenzl@redhat.com>
Date: Sun, 29 Aug 2010 15:49:15 -0400
Subject: [block] cciss: Remove unused was_only_controller_node
Message-id: <1283097002-3341-17-git-send-email-thenzl@redhat.com>
Patchwork-id: 27863
O-Subject: [RHEL6 PATCH 16/63] cciss: Remove unused,
	code about was_only_controller_node.
Bugzilla: 568830
RH-Acked-by: Neil Horman <nhorman@redhat.com>

Removed some unused, superflous code about was_only_controller_node.

diff --git a/drivers/block/cciss.c b/drivers/block/cciss.c
index 022631d..155460b 100644
--- a/drivers/block/cciss.c
+++ b/drivers/block/cciss.c
@@ -1752,7 +1752,6 @@ static void cciss_update_drive_info(int ctlr, int drv_index, int first_time)
 	unsigned long flags = 0;
 	int ret = 0;
 	drive_info_struct *drvinfo;
-	int was_only_controller_node;
 
 	/* Get information about the disk and modify the driver structure */
 	inq_buff = kmalloc(sizeof(InquiryData_struct), GFP_KERNEL);
@@ -1760,13 +1759,6 @@ static void cciss_update_drive_info(int ctlr, int drv_index, int first_time)
 	if (inq_buff == NULL || drvinfo == NULL)
 		goto mem_msg;
 
-	/* See if we're trying to update the "controller node"
-	 * this will happen the when the first logical drive gets
-	 * created by ACU.
-	 */
-	was_only_controller_node = (drv_index == 0 &&
-				h->drv[0].raid_level == -1);
-
 	/* testing to see if 16-byte CDBs are already being used */
 	if (h->cciss_read == CCISS_READ_16) {
 		cciss_read_capacity_16(h->ctlr, drv_index, 1,
